### PR TITLE
feat:delete and edit post

### PR DESCRIPTION
**Pull request by tushargoyal812**
https://user-images.githubusercontent.com/69681896/168157815-c9a2246a-0f54-4c61-8ab1-e4b45a8cb049.mp4

### CODE REVIEW

File: my-tweet/src/components/post-menu.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    Icon,useDisclosure
+  } from '@chakra-ui/react'
+  import { GoKebabVertical } from "react-icons/go";
+import { usePost } from '../context/post-context';
+import { EditModal } from './edit-modal';
+export const PostMenu=({userPost})=>{
+    const{setPosts,posts}=usePost()
+    const { isOpen, onOpen, onClose } = useDisclosure()
+
+    const deletePostHandler=(id)=>{
```
Comment (deepak29-git) [inline my-tweet/src/components/post-menu.js:20]: use api for delete operation

File: my-tweet/src/pages/Sign-up.js [hunk 2]
```diff
@@ -55,6 +56,7 @@ export const SignUp=()=>{
       <InitialFocus isOpen={isOpen} onClose={onClose} />
       </FormControl>
       </Box>
+      {/* <Toast/> */}
```
Comment (deepak29-git) [inline my-tweet/src/pages/Sign-up.js:59]: remove commented code if it is not used

**Review by deepak29-git — APPROVED** [review]

**Review by sharmayatendra — APPROVED** [review]
code looks clean
design is consistent
file structure is also good

File: my-tweet/src/components/post-menu.js (hunk 1 above)
Comment (tushargoyal812) [reply my-tweet/src/components/post-menu.js:20]: ok will do it

File: my-tweet/src/pages/Sign-up.js (hunk 2 above)
Comment (tushargoyal812) [reply my-tweet/src/pages/Sign-up.js:59]: ok

**Pull request merged**